scripts/gh-pr-assigned.coffee

```diff
@@ -18,11 +21,24 @@
 #
 # Authors:
 #   ram535ii
+#   Thanks to https://github.com/hubot-scripts/hubot-github-repo-event-notifier for inspiration
+
+room = process.env["HUBOT_GITHUB_EVENT_NOTIFIER_ROOM"]
+user = process.env["HUBOT_GITHUB_USERNAME"]
+action = "assigned"
+
+if !room?
+  console.warn "A room must be specific for posting to"
+
+if !user?
+  console.warn "A user to check for pull request assignment must be specified"
```

Comment:
I'd recommend including the name of the env variable that needs to be set in the warnings here, makes it that teensy bit more useful when someone hits it